devenv/README.md

```diff
@@ -0,0 +1,78 @@
+### Windows Dev Env
+
+This folder contains a few scripts to help get setup with a Windows Development Environment on a Mac.
```

Comment:
This mentions `on a Mac` but the first step is to run a powershell script. Should the powershell stuff be after setting up the vagrant box or is the powershell stuff meant for Windows and the vagrant box for Mac?

Reply:
Can you add that to the titles? Eg: `If you have a Windows machine or VM already` and `If you need a Windows VM`. And maybe then remove the `on Mac` part since the instructions can be followed on a real Windows box?